### ISSUE TITLE
MySQL backend

### ISSUE THREAD

**Issue by fubu** [amiv-eth/amivapi#4]
The app will run on MySQL in production, so add support for it. Since we are already using SQLAlchemy for the data layer, adding MySQL support is as simple as switching the SQLALCHEMY_DATABASE_URI setting to a MySQL URI. The real issue is to support multiple instances running seperated from each other on the same database, e.g. when running the test suite on the developer machine (or even multiple test suites in parallel).

In short: Make sure the tests create their own (preferably random) database "on-the-fly".
